src/include/catalog/manager.h

```diff
@@ -21,13 +21,13 @@
 
 #include "common/macros.h"
 #include "common/types.h"
-#include "container/cuckoo_map.h"
 #include "container/lock_free_array.h"
 
 namespace peloton {
 
 namespace storage {
 class TileGroup;
+class IndirectionArray;
 }
 
 namespace catalog {
```

Comment:
Actually, refactor the name of Manager to TileManager or TileGroupManager to make it easier to understand.